internal/collector/metrics.go

```diff
@@ -0,0 +1,106 @@
+package collector
+
+import (
+	"fmt"
+	"github.com/shirou/gopsutil/v3/mem"
```

Comment:
goimports && gofumpt!!!

Reply:
[add golangci-lint to github actions](https://github.com/sreway/yametrics/actions/runs/2721401758)